src/App.jsx

```diff
@@ -168,18 +90,18 @@ export default function App() {
 	return (
 		<div className="App">
 			<h1>Checkbox React App</h1>
-			{data.map((cls) => (
+			{data.map((classObj, classIndex) => (
 				<ClassCheckBox
-					key={cls.id}
```

Comment:
can we use props.children here to render it and avoid this much props being passed here?.